File: POOPHP/app/Model/AlumnoCurso.php
<?php
namespace App\Model;
class AlumnoCurso
{
    public $idAlumno="";
    public $idCurso="";
    public $Nota="";
    public $Periodo="";

    public function __construct(String $idalu,String $idcu,int $not,String $peri)
    {
        $this->establecerDatos($idalu,$idcu,$not,$peri);
    }
    public function ObteneridAlumno()
    {
        return $this->idAlumno;
    }
    public function ObteneridCurso()
    {
        return $this->idCurso;
    }
    public function ObtenerNota()
    {
        return $this->Nota;
    }
    public function ObtenerPeriodo()
    {
        return $this->Periodo;
    }
    public function establecerDatos(String $idalum,String $idcur,int $noti,String $perio)
    {
        $this->idAlumno=$idalum;
        $this->idCurso=$idcur;
        $this->Nota=$noti;
        $this->Periodo=$perio;
    }
}
?>

File: POOJAVA/src/clasesPOO/AlumnoCurso.java
package clasesPOO;

public class AlumnoCurso 
{
    String idAlumno;
    String idCurso;
    int Nota;
    String Periodo;

    public AlumnoCurso(String idalu,String idcu,int not,String peri)
    {
        establecerDatos(idalu,idcu,not,peri);
    }
    public String ObteneridAlumno()
    {
       return idAlumno;
    }
    public String ObteneridCurso()
    {
       return idCurso;
    }
    public int ObtenerNota()
    {
       return Nota;
    }
    public String ObtenerPeriodo()
    {
       return Periodo;
    }
    void  establecerDatos(String idalum,String idcur,int noti,String perio)
    {
        idAlumno=idalum;
        idCurso=idcur;
        Nota=noti;
        Periodo=perio;
    }
}
